shepherd_bms/src/compute.cpp

```diff
@@ -15,25 +15,27 @@ void ComputeInterface::enableCharging(bool enableCharging){
     isChargingEnabled = enableCharging;
 }
 
-FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint16_t currentToSet)
+FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint16_t currentToSet, uint8_t state_of_charge)
 {
     if (!isChargingEnabled)
     {
         chargerMsg.cfg.chargerControl = 0b101;
         sendMessageCAN2(CANMSG_CHARGER, 8, chargerMsg.msg);
-        Serial.println("DISABLED!");
         //return isCharging() ? FAULTED : NOT_FAULTED; //return a fault if we DO detect a voltage after we stop charging
         return NOT_FAULTED;
     }
 
     // equations taken from TSM2500 CAN protocol datasheet
     chargerMsg.cfg.chargerControl = 0xFC;
     chargerMsg.cfg.chargerVoltage = voltageToSet * 10;
-    if (currentToSet > 10) {
+    if (currentToSet > 10) 
+    {
         currentToSet = 10;
     }
     chargerMsg.cfg.chargerCurrent = currentToSet * 10 + 3200;
-    chargerMsg.cfg.chargerLEDs = 0x01;
+    bool is_charging = isCharging();
```

Comment:
This function shouldn't be used here. We should be checking if there is current, not if we are in charge mode. We will always be in charge mode when the charger is on so it will always blink.

@dyldonahue we should rename "isCharging" to like "isChargerConnected" and also update the code to use it in the conditional for the charging state